posthog/migrations/0101_org_owners.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.0.6 on 2020-11-19 06:04
+
+from django.db import migrations, models
+
+
+def make_first_administrators_owners(apps, schema_editor):
+    Organization = apps.get_model("posthog", "Organization")
+    for organization in Organization.objects.all():
+        first_admin = organization.memberships.filter(level=8).first()
```

Comment:
Should we add a created_at filter here?

Reply:
Maybe not a `created_at` one, but a `level=15` one.